nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Mb employee details

# Description

Closes #17 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout mb-employee-details`
4. Using TablePlus/dBeaver, insert additional data into your trainingemployee table with the following queries:
```
INSERT INTO hrapp_trainingprogramemployee VALUES (NULL, 1, 2);
INSERT INTO hrapp_trainingprogramemployee VALUES (NULL, 1, 3);
INSERT INTO hrapp_trainingprogramemployee VALUES (NULL, 1, 4);
```
5. Check that the employee details have their associated names, departments, computers + training programs (first employee should have multiple training programs).

# Checklist:
